2022-08-31 16:08:34,232 INFO    MainThread:4911 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/.config/wandb/settings
2022-08-31 16:08:34,232 INFO    MainThread:4911 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/leo/Cloud-Classification/cloud_classification/wandb/settings
2022-08-31 16:08:34,232 INFO    MainThread:4911 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-08-31 16:08:34,232 INFO    MainThread:4911 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'cloud_classification/train.py', 'program': '/home/ltorres/leo/Cloud-Classification/cloud_classification/train.py'}
2022-08-31 16:08:34,233 INFO    MainThread:4911 [wandb_init.py:_log_setup():405] Logging user logs to /home/ltorres/leo/Cloud-Classification/cloud_classification/wandb/run-20220831_160834-p2lgy8uq/logs/debug.log
2022-08-31 16:08:34,233 INFO    MainThread:4911 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ltorres/leo/Cloud-Classification/cloud_classification/wandb/run-20220831_160834-p2lgy8uq/logs/debug-internal.log
2022-08-31 16:08:34,233 INFO    MainThread:4911 [wandb_init.py:init():439] calling init triggers
2022-08-31 16:08:34,233 INFO    MainThread:4911 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {'hardware': {'device': 'cuda:2'}, 'data': {'path_dataset': '/data/ltorres', 'path_save_weights': '/data/ltorres/CLOUD_CLASSIFICATION_WEIGHTS', 'path_save_logs': '/data/ltorres/model_logs', 'class_names': ['1_cumulus', '2_altocumulus', '3_cirrus', '4_clearsky', '5_stratocumulus', '6_cumulonimbus', '7_mixed'], 'resize': 256, 'use_augmentation': True}, 'hyperparameters': {'epochs': 30, 'batch_size': 32, 'learning_rate': 5e-05, 'early_stopping_tolerance': 8, 'criterion': 'cross_entropy', 'optimizer': 'sgd', 'use_scheduler': True, 'lr_decay_steps': 15, 'lr_decay_gamma': 0.5}, 'model': {'hidden_dim': 512, 'num_hidden': 2, 'num_classes': 7, 'feature_extraction': False, 'conv_type': 'gcn', 'conv_parameters': {'num_heads': 2, 'agg': 'sum'}, 'gnn_dropout': 0.3, 'adjacency_builder': 'cos_sim', 'builder_parameter': 0.5, 'use_both_heads': False, 'loss_lambda': 1}}
2022-08-31 16:08:34,233 INFO    MainThread:4911 [wandb_init.py:init():492] starting backend
2022-08-31 16:08:34,233 INFO    MainThread:4911 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-31 16:08:34,247 INFO    MainThread:4911 [backend.py:ensure_launched():219] starting backend process...
2022-08-31 16:08:34,253 INFO    MainThread:4911 [backend.py:ensure_launched():224] started backend process with pid: 4933
2022-08-31 16:08:34,255 INFO    MainThread:4911 [wandb_init.py:init():501] backend started and connected
2022-08-31 16:08:34,263 INFO    MainThread:4911 [wandb_init.py:init():565] updated telemetry
2022-08-31 16:08:34,281 INFO    MainThread:4911 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-08-31 16:08:35,082 INFO    MainThread:4911 [wandb_run.py:_on_init():1759] communicating current version
2022-08-31 16:08:35,159 INFO    MainThread:4911 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.13.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-31 16:08:35,159 INFO    MainThread:4911 [wandb_init.py:init():625] starting run threads in backend
2022-08-31 16:08:36,749 INFO    MainThread:4911 [wandb_run.py:_console_start():1733] atexit reg
2022-08-31 16:08:36,750 INFO    MainThread:4911 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-08-31 16:08:36,751 INFO    MainThread:4911 [wandb_run.py:_redirect():1611] Redirecting console.
2022-08-31 16:08:36,754 INFO    MainThread:4911 [wandb_run.py:_redirect():1667] Redirects installed.
2022-08-31 16:08:36,754 INFO    MainThread:4911 [wandb_init.py:init():664] run started, returning control to user process
2022-08-31 16:08:39,722 INFO    MainThread:4911 [wandb_watch.py:watch():47] Watching
2022-08-31 16:08:42,105 INFO    MainThread:4911 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 255
2022-08-31 16:08:42,110 INFO    MainThread:4911 [wandb_run.py:_restore():1674] restore
